### PR TITLE
Update install doc

### PR DESCRIPTION
**Pull request by IanMadd**
## Related Issue

- #7129 
- #7118

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs-chef-io/content/inspec/install.md [hunk 1]
```diff
@@ -21,12 +21,14 @@ To see which platforms and platform versions Chef InSpec is supported on, see th
...
 
-{{< note >}}Please follow steps mentioned in this document [Chef Software Install Script](/install_omnibus/). Please replace `<LICENSE_ID>` with your licenseId.{{< /note >}}
-
 ```bash
 curl -L https://chefdownload-commerical.chef.io/install.sh?license_id=<LICENSE_ID> | sudo bash -s -- -P inspec
 ```
 
+Replace `<LICENSE_ID>` with your license ID.
+
+For more information about the install script, see the [Chef Install Script documentation](/chef_install_script/).
```
Comment (Vasu1105) [inline docs-chef-io/content/inspec/install.md:30]: So, the install script link is currently giving a 404 error so that is expected right @IanMadd

Comment (IanMadd) [reply docs-chef-io/content/inspec/install.md:30]: @Vasu1105 
Where is it sending you? 
It works for me --> https://deploy-preview-7144--chef-inspec.netlify.app/chef_install_script/

Comment (Vasu1105) [reply docs-chef-io/content/inspec/install.md:30]: It's landing me here https://github.com/inspec/inspec/blob/d2e6dc27791ecca78d237650935a4cb780182945/chef_install_script

Comment (IanMadd) [reply docs-chef-io/content/inspec/install.md:30]: The chef_install_script page comes from another repo, so it won't appear here. 
Try here https://deploy-preview-7144--chef-inspec.netlify.app/inspec/install/

Comment (Vasu1105) [reply docs-chef-io/content/inspec/install.md:30]: Ok. Merging the PR. Thanks!!

**Pull request merged**